src/garage/tf/algos/ddpg.py

```diff
@@ -58,6 +60,10 @@ class DDPG(OffPolicyRLAlgorithm):
 
     """
 
+    # pylint: disable=too-many-instance-attributes
+    # pylint: disable=too-many-arguments
```

Comment:
please don't do whole-file disables. if you think we should globally turn off some rules, let's talk about it in the comments.

Reply:
Ah..you are right. I didn't intend to do it globally.